src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -196,4 +196,31 @@ public void testDoPost_CleansHtmlContent() throws IOException, ServletException
 
     Mockito.verify(mockResponse).sendRedirect("/chat/test_conversation");
   }
+  
+  @Test
+  public void testDoPost_ModifyMessageBoldItalics() throws IOException, ServletException{
+    Mockito.when(mockRequest.getRequestURI()).thenReturn("/chat/test_conversation");
+    Mockito.when(mockSession.getAttribute("user")).thenReturn("test_username");
+
+    User fakeUser = new User(UUID.randomUUID(), "test_username", "password", Instant.now());
```

Comment:
Might have to add a new field depending on whether Tishauna merges ahead or later.